### ISSUE TITLE
Fedora Chat refused to send messages until I logged out and in again. I could see other people's messages

### ISSUE THREAD

**Issue by mattdm** [vector-im/element-web#19718]
### Steps to reproduce

1. Where are you starting? What can you see?

In the #fedora-meeting:fedora.im channel on https://chat.fedoraproject.org/

2. What do you click?

Edit message. Then after edit, "Enter" and "Save" are unresponsive.

This was today at around 19:02 

Clicked cancel, and found I was unable to send any messages.

3. More steps…

Noticed that my phone was working fine. Logged out and in to web client again and now it works

#### What did you expect?

Messages sent

#### What happened instead?

Messages not sent.

### Operating system

Fedora Workstation

### Browser information

Firefox 94.0

### URL for webapp

https://chat.fedoraproject.org/

### Application version

Fedora Chat version: v1.9.4 Olm version: 3.2.3

### Homeserver

fedora.im

### Will you send logs?

Yes

**Comment by jaller94** [comment]
I don't see the room alias #fedora-meeting:fedora.im.
Do you mean the public room #meeting:fedoraproject.org?

**Comment by mattdm** [comment]
yes, sorry. the room is `#fedora-meeting` on IRC and I am not yet used to thinking of it that way.

**Comment by jaywink** [comment]
Hi @mattdm . Could you clarify whether;

1) the time you gave is UTC or something else?
2) whether you know of other users who had the issue? This would help figure out whether this looks like a homeserver issue or a client issue.

Thanks!

**Comment by dbkr** [comment]
It looks like an event has got stuck failing to send and then other messages have got queued up behind it. I can't tell what that first event was though.

**Comment by mattdm** [comment]
UTC, yes.

To my knowledge, it was just me. But I swear it really happened. :)

**Comment by jaywink** [comment]
> It looks like an event has got stuck failing to send and then other messages have got queued up behind it. I can't tell what that first event was though.

@dbkr From the logs I believe the first transaction that failed to reach the homeserver was m1636657375810.66. This I believe was an edit of m1636657358755.65 which was ack'd by the homeserver.

**Comment by mattdm** [comment]
Is m1636657358755.65  `VERY AGGRESISVE TICKET CLEANUP`? That would match with what I experienced -- it got "stuck" trying to edit that.

**Comment by jaywink** [comment]
Have not looked at message contents, but it matches the event ID you sent separately :+1:
